torchvision/prototype/transforms/_presets.py

```diff
@@ -41,7 +41,7 @@ def forward(self, img: Tensor) -> Tensor:
             img = F.pil_to_tensor(img)
         img = F.convert_image_dtype(img, torch.float)
         img = F.normalize(img, mean=self._mean, std=self._std)
-        return img
+        return cast(Tensor, img)
```

Comment:
Why is this necessary? The return type of the functional API is `Tensor` so this casting seems unnecessary. Here is how normalize looks for example:
```python
def normalize(tensor: Tensor, mean: List[float], std: List[float], inplace: bool = False) -> Tensor:
```

Reply:
The problem is that the decorator is annotated with

```py
def log_api_usage_once(fn: Callable) -> Callable:
```

This means `mypy` assumes `Callable` as full annotation for the decorated function. This is roughly equivalent to

```py
def foo(*args: Any, **kwargs: Any) -> Any
```

If we restrict the decorator for the functional transforms for now, we can fix this by annotating with `Callable[..., torch.Tensor]`.

Reply:
Thanks @pmeier, makes sense. No need to change then.